scripts/fix-shrinkwrap.js

```diff
@@ -52,36 +62,38 @@ function fixup(dependency) {
 
   return false;
 
-}
+};
 
-// Walk the shrinkwrap dependency tree,
-// and fix up any mangled versions along the way
-// while counting how many have been fixed (because, why not?)
-function walk(tree) {
+/**
+ * Walk the shrinkwrap dependency tree,
+ * and fix up any mangled versions along the way
+ * while counting how many have been fixed (because, why not?)
+ * @param {Object} tree - dependency tree
+ * @returns {Number}
+ */
+const walk = (tree) => {
 
   if (!tree.dependencies) {
-    return 0; // Nothing to do here
+    // Nothing to do here
+    return ZERO;
   }
 
-  var keys = Object.keys(tree.dependencies);
-  var fixes = 0;
-  var dep = null;
+  let fixes = ZERO;
 
-  for (var i = 0; i < keys.length; i++) {
-    dep = tree.dependencies[keys[i]];
-    fixes += fixup(dep) ? 1 : 0;
```

Comment:
I guess `fixes += fixup(dep) ? 1 : 0;` could alternatively be replaced by `fixes += Number(fixup(dep))` ?